### DISCUSSION TITLE
What directory are the thumbnails stored when enabling "Use optimized images in the thumbnail interface?"

### DISCUSSION THREAD

**Discussion by trebory6** [AlUlkesh/stable-diffusion-webui-images-browser#236] [Q&A] [answered]
Hey guys,

Was hoping someone knew where the low resolution thumbnails for this extension was stored when you enable the "Use optimized images in the thumbnail interface" 

I can't seem to find anything in the extension directory or the output directory. 

I am on Windows as well.

**Comment by AlUlkesh** [comment, marked as answer]
They're stored in a sub-directory called _optimized_

And that one is either in the directory you specified in `Settings - Saving images/grids - Directory for temporary images`

or if you left this empty in your system temp dir, (cd %temp%).
